packages/vsphere-csi/charts/Chart.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: v1
+name: vsphere-csi
+version: 2.1.0
+appVersion: 2.1.0
+description: vSphere CSI driver
+sources:
+  - https://github.com/kubernetes-sigs/vsphere-csi-driver
+icon: https://charts.rancher.io/assets/logos/vsphere-csi.svg
+maintainers:
+  - name: Rancher
+    email: caleb@rancher.com
```

Comment:
Should we be using SUSE email addresses and should this be Steven?

Reply:
We discussed this and we will make a generic mailing list in the future for this, but won't change it for this release. Lucky @cbron